src/Sulu/Bundle/ActivityBundle/UserInterface/Controller/ActivityController.php

```diff
@@ -383,7 +383,7 @@ private function getActivityDescription(array $activity, string $translationLoca
             : $this->translator->trans('sulu_activity.someone', [], 'admin', $translationLocale);
 
         $translationParameters['{resourceTitle}'] = $this->getLocalizedValue(
-            $activity['resourceTitle'],
+            $activity['resourceTitle'] ?? '',
```

Comment:
maybe we should adjust the typehint of the `getLocalizedValue` instead of passing an empty string here? 🤔

Reply:
updated!